LPS-172219 Fix POSHI automation | Master

alessandromalves opened the pull request:
This is only for master
https://issues.liferay.com/browse/LPS-172219
cc.: @david-gutierrez-mesa @Tim-Cao

liferay-echo commented:
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

alessandromalves commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-source-format/4683/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/4683/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/22bb9a8e4bb9147e1b492299fa4283b77a5145f3">22bb9a8e4bb9147e1b492299fa4283b77a5145f3</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/alessandromalves/liferay-portal/tree/LPS-172219">LPS-172219</a><br/>Branch GIT ID: <a href="https://github.com/alessandromalves/liferay-portal/commit/a6fd4370e86bd2e71ea47e217ccaa4559373c136">a6fd4370e86bd2e71ea47e217ccaa4559373c136</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-source-format/4683/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/4683/jenkins-report.html">here</a>.</h5></details></html>

alessandromalves commented:
Hey @Tim-Cao, I also updated the two tests in addition to updating the paths. I removed the `AssertVisible` function from the tests because I saw that an assert was performed inside `Uncheck.uncheckNotVisible`. I also added `Staging.publishToLive();` in place of the other functions.

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/4683/">test-portal-source-format#4683</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/4683/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/10800">liferay-echo#10800</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2152161226">[master] ci:test:sf - alessandromalves &gt; liferay-echo - PR#10800 - 2023-01-06[14:46:16]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/17896/">publish-testray-report#17896</a><br/></div></div>

Tim-Cao commented on portal-web/test/functional/com/liferay/portalweb/paths/portlet/staging/StagingPublishToLive.path:
```diff
@@ -123,12 +123,12 @@
 <tr>
 	<td>PAGES_TO_PUBLISH_CHECKBOX_UNCHECKED</td>
-	<td>//div[@class='pages-selector']//div[normalize-space()='${key_entryName}' and not(contains(@class,'checked'))]//span[contains(@class,'icon-check')]</td>
+	<td>//span[normalize-space(text())='${key_entryName}']//ancestor::div[@class='autofit-row']//input[@type='checkbox' and not(@checked)]</td>
```
`//div[contains(@class,'layout-selector')]//div[contains(@class,'treeview-link')][*//div[normalize-space()='${key_entryName}']]//input[@type='checkbox'][not(@checked)]`

Tim-Cao reviewed (commented):
Hi @alessandromalves, would you mind modifying according to comments? Thanks

alessandromalves commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-38.liferay.com/job/test-portal-source-format/3574/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/3574/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f432b9e8a6df4a1975f64544b8fff8af539e1291">f432b9e8a6df4a1975f64544b8fff8af539e1291</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/alessandromalves/liferay-portal/tree/LPS-172219">LPS-172219</a><br/>Branch GIT ID: <a href="https://github.com/alessandromalves/liferay-portal/commit/70607518abd40c5e475f8599375c7b3204e021b3">70607518abd40c5e475f8599375c7b3204e021b3</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-38.liferay.com/job/test-portal-source-format/3574/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/3574/jenkins-report.html">here</a>.</h5></details></html>

alessandromalves commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
`ci:test:relevant`
`ci:test:sf`

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-39.liferay.com/job/forward-pullrequest/214//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/128490
[Console](https://test-1-39.liferay.com/job/forward-pullrequest/214//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-38.liferay.com/job/test-portal-source-format/3574/">test-portal-source-format#3574</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/3574/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/10800">liferay-echo#10800</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2153244613">[master] ci:test:sf - alessandromalves &gt; liferay-echo - PR#10800 - 2023-01-09[03:32:10]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-38.liferay.com/job/publish-testray-report/12449/">publish-testray-report#12449</a><br/></div></div>
